gflownet/gflownet.py

```diff
@@ -202,7 +202,7 @@ def sample_actions(
         sampling_method: Optional[str] = "policy",
         backward: Optional[bool] = False,
         temperature: Optional[float] = 1.0,
-        random_action_prob: Optional[float] = 0.0,
+        random_action_prob: Optional[float] = None,
         no_random: Optional[bool] = True,
         times: Optional[dict] = None,
```

Comment:
By the way, `times` argument is never used in the method.

Reply:
Yeah, I am aware. At some points I discontinued the support of calculating and logging times, but I would like to pick it up again at some point.